osu.Game/Screens/Select/FooterButton.cs

```diff
@@ -6,183 +6,234 @@
-                            Children = new Drawable[]
+                            Top = 12,
+                            Right = shear_padding,
+                        },
+                        Children = new Drawable[]
+                        {
+                            sprite = new SpriteIcon
                             {
-                                TextContainer = new Container
+                                Shear = -SHEAR,
+                                Anchor = Anchor.TopCentre,
+                                Origin = Anchor.TopCentre,
+                                Size = new Vector2(20)
+                            }
+                        }
+                    },
+                    ButtonContentContainer = new FillFlowContainer
+                    {
+                        Anchor = Anchor.TopCentre,
+                        Origin = Anchor.TopCentre,
+                        Direction = FillDirection.Horizontal,
+                        AutoSizeAxes = Axes.X,
+                        Children = new Drawable[]
+                        {
+                            TextContainer = new Container
+                            {
+                                Colour = Colour4.White,
+                                Anchor = Anchor.TopCentre,
+                                Origin = Anchor.TopCentre,
+                                Shear = -SHEAR,
+                                Margin = new MarginPadding { Top = 41.89f },
+                                Padding = new MarginPadding { Right = 14 },
+                                AutoSizeAxes = Axes.Both,
+                                Child = spriteText = new OsuSpriteText
                                 {
+                                    Font = TorusFont,
+                                    AlwaysPresent = true,
                                     Anchor = Anchor.Centre,
                                     Origin = Anchor.Centre,
-                                    AutoSizeAxes = Axes.Both,
-                                    Child = SpriteText = new OsuSpriteText
+                                }
+                            }
+                        }
+                    },
+                    new Container
+                    {
+                        Anchor = Anchor.TopCentre,
+                        Origin = Anchor.TopCentre,
+                        Padding = new MarginPadding
+                        {
+                            Top = 77,
+                        },
+                        Children = new Drawable[]
+                        {
+                            new Container
+                            {
+                                CornerRadius = 3,
+                                Anchor = Anchor.TopCentre,
+                                Origin = Anchor.TopCentre,
+                                Height = 6,
+                                Width = 120.74f,
```

Comment:
Again, this is way too (incorrectly) precise.

Reply:
![size](https://user-images.githubusercontent.com/74463310/188597170-e3b32a25-7568-4f7d-81f9-72e27963dfe3.png)
shall i go with 120 then?

Reply:
I'd probably have made it relative X + padding, else it won't expand properly if the button is made to grow wider.